src/BasicModelInterface/Logger.cs

```diff
@@ -1,9 +1,9 @@
-﻿using System.Runtime.InteropServices;
+using System.Runtime.InteropServices;
 using System.Text;
 
 namespace BasicModelInterface
 {
-    public enum Level { Off, Debug, Info, Error, Fatal };
+    public enum Level { LEVEL_ALL, LEVEL_DEBUG, LEVEL_INFO, LEVEL_WARNING, LEVEL_ERROR, LEVEL_FATAL, LEVEL_NONE };
```

Comment:
I'd better call them: public enum Level { All, Debug, Info, Warning, Error, Fatal, None }. This way it is still compatible with the C/C++ version, but also with the CSharp naming conventions: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions.